src/_locales/en/messages.json

```diff
@@ -188,107 +188,107 @@
   },
   "turnOnProtection": {
     "message": "Turn on protection",
-    "description": "Button label"
+    "description": "Button label."
   },
   "removeCustomLocation": {
     "message": "Remove custom location",
-    "description": "Button label"
+    "description": "Button label."
   },
   "websiteProtection": {
     "message": "Website protection",
-    "description": "A subtitle that is followed by an off icon and the word 'Off'"
+    "description": "A subtitle that is followed by an off icon and the word 'Off'."
   },
   "giveFeedback": {
     "message": "Give feedback",
-    "description": "Button label"
+    "description": "Button label."
   },
   "learnMore": {
     "message": "Learn more",
-    "description": "Label of link that leads to SUMO"
+    "description": "Label of link that leads to SUMO."
   },
   "onboarding1_title": {
     "message": "Turn off VPN for specific websites",
-    "description": "Title in onboarding panel"
+    "description": "Title in onboarding panel."
   },
   "onboarding1_body": {
     "message": "With the Firefox extension, you can turn off VPN protection for specific websites, while keeping it on for everything else.",
-    "description": "Body text in onboarding panel"
+    "description": "Body text in onboarding panel."
   },
   "onboarding2_title": {
     "message": "See different websites from different locations",
-    "description": "Title in onboarding panel"
+    "description": "Title in onboarding panel."
   },
   "onboarding2_body": {
-    "message": "",
-    "description": "Body text in onboarding panel"
+    "message": "News from London. Sports from Sao Paulo. Shopping from Tokyo. Set website-specific locations for personalized browsing.",
```

Comment:
I assume this should be the content based on Figma

Reply:
Thank you!